projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/jupyter_sample_job/10_notebook.ipynb

```diff
@@ -1,35 +1,90 @@
 {
- "cells": [],
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16a6ea87-7142-44dd-94b6-e95fe2d57d06",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "vdk_ipython extension introduces a magic command for Jupyter.\n",
+    "The command enables the user to load VDK for the current notebook.\n",
+    "VDK provides the job_input API, which has methods for:\n",
+    "    * executing queries to an OLAP database;\n",
+    "    * ingesting data into a database;\n",
+    "    * processing data into a database.\n",
+    "See the IJobInput documentation for more details.\n",
+    "Please refrain from tagging this cell with VDK as it is not an actual part of the data job\n",
+    "and is only used for development purposes.\n",
+    "\"\"\"\n",
+    "%reload_ext vdk_ipython\n",
+    "%reload_VDK\n",
+    "job_input = VDK.get_initialized_job_input()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "712a949e-8e9e-4c31-972a-6fab1b274c95",
+   "metadata": {
+    "tags": [
+     "vdk"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "This and the subsequent cells are marked as VDK cells.\n",
+    "\n",
+    "Cells can be marked or unmarked as VDK cells through the gearwheel menu\n",
+    "to the right.\n",
+    "\"\"\"\n",
+    "\n",
+    "job_input.execute_query(\n",
+    "    \"CREATE TABLE IF NOT EXISTS hello_world (id NVARCHAR);\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36137500-9136-4083-8b04-073ed84e2d4c",
+   "metadata": {
+    "tags": [
+     "vdk"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "job_input.send_object_for_ingestion(\n",
+    "    payload=dict(id=\"Hello World!\"),\n",
+    "    destination_table=\"hello_world\"\n",
+    ")"
+   ]
+  }
+ ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
What's this? Why is it chnaged?

Reply:
idk it must have been changed automatically by my jupyterlab instance